### ISSUE TITLE
Difference in result on TRN-pytorch example

### ISSUE THREAD

**Issue by ankitshah009** [metalbubble/moments_models#8]
Hi all, 

For the example mentioned, I am able to observe a slight difference in the result in comparison to what is mentioned on the website. 

python test_video.py --arch InceptionV3 --dataset moments --weight pretrain/TRN_moments_RGB_InceptionV3_TRNmultiscale_segment8_best.pth.tar --frame_folder sample_data/bolei_juggling
Multi-Scale Temporal Relation Network Module in use ['8-frame relation', '7-frame relation', '6-frame relation', '5-frame relation', '4-frame relation', '3-frame relation', '2-frame relation']
Freezing BatchNorm2D except the first one.
Loading frames in sample_data/bolei_juggling
RESULT ON sample_data/bolei_juggling
0.994 -> juggling
0.002 -> flipping
0.001 -> spinning
0.001 -> stacking
0.001 -> drumming

Though, the difference is minor wanted to understand what could be a potential difference as I receive no error while running the example nor the installation packages. I am using python 3.6 with dependencies installed.  

Looking forward to your response. 

Kind Regards,
Ankit Shah

**Comment by ankitshah009** [comment]
Any update on the issue?

**Comment by metalbubble** [comment]
There is no problem for this. It comes from some random sampling of relations in the TRNmodule. Take a look at the TRNmodule.py.
